docs/buildah-build.1.md

```diff
@@ -476,6 +476,10 @@ If the specified capabilities are not in the default set, container engines
 should print an error message and will run the container with the default
 capabilities.
 
+**--label2annotation** *bool-value*
+
+Labels to annotations tells Buildah to save labels defined in the Containerfile as annotations in OCI Images. (Default is `false`).
```

Comment:
It really needs to be clear that the image gets annotations _instead of_ labels.

Reply:
How about now?

Reply:
The first two sentences read kind of weird to me.  The third has a subject/verb agreement problem.